src/Sulu/Component/Content/Document/Behavior/BlameBehavior.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Document\Behavior;
+
+use Sulu\Component\Security\Authentication\UserInterface;
+
+/**
+ * Adds the user who created and lastly changed the document.
+ */
+interface BlameBehavior
+{
+    /**
+     * @return UserInterface
+     */
+    public function getCreator();
+
+    /**
+     * @return UserInterface
+     */
+    public function getChanger();
```

Comment:
What's the point of this method? Why do we need the person that last changed a document? If we do, why is this not called `getLastModifiedBy()` or similar?